src/universal_transfer_operator/data_providers/database/google/bigquery.py

```diff
@@ -181,3 +472,132 @@ def openlineage_dataset_uri(self) -> str:
         https://github.com/OpenLineage/OpenLineage/blob/main/spec/Naming.md
         """
         return f"{self.openlineage_dataset_namespace}{self.openlineage_dataset_name}"
+
+
+class S3ToBigqueryDataTransfer:
+    """
+    Create and run Datatransfer job from S3 to Bigquery
+    :param source_file: Source file that is used as source of data
+    :param target_table: Table that will be created on the bigquery
```

Comment:
Maybe we could make the order consistent with `__init__`.

```suggestion
    :param target_table: Table that will be created on the bigquery
    :param source_file: Source file that is used as source of data
```

Reply:
[Done](https://github.com/astronomer/apache-airflow-provider-transfers/pull/66/commits/49dea8f8e568dc9b61f2dbb54d340c4c42f6ead1)